spec/models/game_result_spec.rb

```diff
@@ -0,0 +1,26 @@
+require 'rails_helper'
+
+RSpec.describe GameResult, type: :model do
+  it "is valid when produced by a factory" do
+    result = FactoryBot.build(:game_result)
+    result.save
+
+    expect(result).to be_valid
+  end
+
+  it "fails validation with non-number scores" do
+    result = FactoryBot.build(:game_result, score_home: "2a")
```

Comment:
You might also check for the existence of a `score_home` in the errors dict after calling `is_valid?`

```
r = FactoryBot.build(:game_result, score_home: "2a")
expect {r.is_valid?}.to change { r.errors }.from([]).to([score_home: [...]])
# OR
expect {r.is_valid?}.to change { r.errors[:score_home].count }.from(0).to(1)
```